src/Pages/MembershipPlan/MembershipPlan.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import './App.css'
+import './MembershipPlan.css'
```

Comment:
```suggestion
import './membership-plan.css'
```
[css file naming conventions](https://www.lifewire.com/naming-css-style-sheet-files-3466881)